go/vt/vtgate/engine/routing.go

```diff
@@ -157,6 +161,14 @@ func (rp *RoutingParameters) findRoute(ctx context.Context, vcursor VCursor, bin
 		default:
 			return rp.in(ctx, vcursor, bindVars)
 		}
+	case Between:
+		switch rp.Vindex.(type) {
+		case vindexes.SingleColumn:
+			return rp.between(ctx, vcursor, bindVars)
+		default:
+			// Only SingleColumn vindex supported.
+			return nil, nil, vterrors.Errorf(vtrpcpb.Code_INTERNAL, "opcode: %v not supported", rp.Opcode)
+		}
```

Comment:
This should use vterrors.VT13001 i.e. Internal Bug, if we ended up selecting this opcode.

Reply:
thanks , incorporated with 40d041f28f5b371854b4538ccc915b0a32f0b79d